modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -782,9 +782,15 @@ class CV_EXPORTS_W VideoCapture
      */
     CV_WRAP String getBackendName() const;
 
+    /**
+     * raise Error::StsError if a method is not successful
+     */
+    CV_WRAP void exceptions(bool enable) { errorOnFail = enable; }
```

Comment:
```
CV_WRAP void setExceptionsMode(bool enable) { errorOnFail = enable; }
CV_WRAP bool getExceptionsMode() { return errorOnFail; }
```

`errorOnFail` => `enableExceptions` / `allowExceptions`

Reply:
tried to follow `fstream` here: https://en.cppreference.com/w/cpp/io/basic_ios/exceptions

but I guess using get/ set is more consistent here.

Reply:
```
CV_WRAP void exceptions(bool enable) { errorOnFail = enable; }
CV_WRAP bool exceptions() const { return errorOnFail; }
```

This work for C++ well, but OpenCV should support bindings like Python/Java.
So get/set is more consistent and straight-forward way to do that.

(we can use `CV_WRAP_AS(...)` here but it would lead for code inconsistency between C++ and Python/Java)